.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+<!--
+This issues database is used to keep track of bugs and new features.
+For questions and support, please visit the Gitter channel instead:
+  https://gitter.im/Vincit/objection.js
+-->
```

Comment:
I think questions and support can still be handled in the issues. They get lost pretty fast in gitter.

Reply:
They should be tagged as questions and in the future they will receive less attention, since the amount of issues is growing.

Reply:
Yes I agree, but I don't think people who post issues can tag them by themselves? The first question could be "what kind of issue is this" similar to how yarn handles it:

https://github.com/yarnpkg/yarn/issues/new